CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Support `ci:php:lint` on Windows
   ([#740](https://github.com/MyIntervals/emogrifier/issues/740),
   [#780](https://github.com/MyIntervals/emogrifier/pull/780))
+- Check the code with psalm
+  ([#537](https://github.com/MyIntervals/emogrifier/issues/537),
+  [#778](https://github.com/MyIntervals/emogrifier/pull/778))
```

Comment:
We currently have a policy of adding new CHANGELOG entries to the top of the relevant section.  (I was caught out by this in my first contribution to the project.)

(Whether that is the best policy is another issue.  The way we're doing things currently, it does mean that the list of changes/fixes in the release note ends up being in reverse-chronological order, which does seem natural to me.)

Reply:
The thinking behind the reverse-chronological order is that then the order of the releases (newest on top) is the same as the order of the changes within the release.